frontend/VolunteerOne/screens/Friends/index.js

```diff
@@ -15,14 +17,10 @@ class ViewFriendsPage extends React.Component {
       <ScrollView
         showsVerticalScrollIndicator={false}
         contentContainerStyle={styles.notifications}>
-        
-        {/* map all notifications using loop */}
-        <Block flex>
-          {profiles.map(profile => (
-             <ProfileItem item={profile} horizontal />
-          ))}
-        </Block>
-    
+          
+      <NewAnouncementModal></NewAnouncementModal>
```

Comment:
Make sure to remove these lines when resolving merge conflicts

Reply:
removed